src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs

```diff
@@ -57,6 +57,13 @@ protected override void OnApplyTemplate()
 			_isTemplateApplied = true;
 
 			UpdateProperties();
+
+#if __WASM__ || __SKIA__
+			//TODO: Workaround for https://github.com/unoplatform/uno/issues/5144
```

Comment:
Apprentely there is common comment prefix `// Uno workaround:` for this kind of stuff.